app/graphql/resolvers/item2item_resolvers.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Optional
+
+import strawberry
+from strawberry.types import Info
+from strawberry import argument
+from strawberry.types import Info
+from typing_extensions import Annotated
+
+from app.data_providers.dispatch import Item2ItemDispatch
+from app.data_providers.item2item import Item2ItemRecommender
+from app.graphql.corpus_recommendation import CorpusRecommendation
+from app.graphql.util import get_field_argument
+from app.singletons import item2item_recommender
+
+DEFAULT_RECOMMENDATION_COUNT = 10
+
+
+async def resolve_syndicated(root, info: Info,
+        resolved_id: Annotated[strawberry.ID, argument(
+            description='Article ID')],
+        recommendation_count: Annotated[Optional[int], argument(
+            description='Maximum number of recommendations to return, defaults to 10')] = 10,
```

Comment:
The `CorpusSlate` already contains a [recomendations count parameter](https://github.com/Pocket/recommendation-api/blob/f5b0da4dd8e7e2c14e95232ccb9f2cf4754263c6/app/graphql/resolvers/corpus_slate_lineup_resolvers.py#L39-L43) on the recommendations field.

Reply:
This is a different API, which is not a part of home. How is it related to CorpusSlate?